### ISSUE TITLE
Change species selection from a drop-down to check-boxs

### ISSUE THREAD

**Issue by SimonGoring** [PalEON-Project/Shiny#9]
If we do that we could add groupings of PFTs, the plotting would be additive, so you could select "Pine, Spruce, Tamarack" or whatever ecological grouping you wanted and it would show the fraction of those taxa.

The new SD would just be the `sqrt(mean(var(sd(taxa))))` although that's not exactly the right formula.  Then, if people want to export we can export a CSV with just those columns, and we can add a checkbox that's 'All Taxa' where it sums to 1 (I'm not sure if the means do actually sum to 1 though).

Or the export could be a set of smaller panels with each taxon, the cumulative plot, and the sd. . .

**Comment by paciorek** [comment]
yes, we could allow groupings of taxa. The new SD would have to be
calculated from the original posterior draws though as that is where the
info on dependence comes from - you can't do it with a formula using the
summary stats.
